app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/AddPictureActivityTest.kt

```diff
@@ -72,12 +70,6 @@ class AddPictureActivityTest {
         BaristaVisibilityAssertions.assertDisplayed(R.id.use_camera)
     }
 
-    @Test
-    fun backButtonWorks() {
```

Comment:
do we know for a fact that the functionality is ok?

Reply:
Only from manual testing. I'll add a couple navigation tests to make sure.

Reply:
Actually there already are some tests in GeneralNavigationTest, so I added only one more.

Reply:
ok I'm happy